service/service.py

```diff
@@ -82,36 +91,21 @@ def download_file(res: requests.Response) -> str:
     :param res: requests.Response instance
     :return: stored file name (path)
     """
-    logging.debug("Chunked file download started")
+    logger.debug("Chunked file download started")
     with tempfile.NamedTemporaryFile(delete=False) as file:
-        for chunk in res.iter_content(chunk_size=CHUNK_SIZE):
+        for chunk in res.iter_content(chunk_size=config.CHUNK_SIZE):
             if chunk:
                 file.write(chunk)
         file.close()
-    logging.debug(f"File stored as {file.name}")
+    logger.debug(f"File stored as {file.name}")
     return file.name
 
 
 if __name__ == "__main__":
-    logging.basicConfig(level=logging.getLevelName(LOG_LEVEL))
 
-    IS_DEBUG_ENABLED = logging.getLogger().isEnabledFor(logging.DEBUG)
+    IS_DEBUG_ENABLED = logger.isEnabledFor(logging.DEBUG)
 
     if IS_DEBUG_ENABLED:
-        APP.run(debug=IS_DEBUG_ENABLED, host='0.0.0.0', port=PORT)
+        APP.run(debug=IS_DEBUG_ENABLED, host='0.0.0.0', port=5000)
     else:
-        import cherrypy
-
-        cherrypy.tree.graft(APP, '/')
-        cherrypy.config.update({
-            'environment': 'production',
-            'engine.autoreload_on': True,
-            'log.screen': False,
-            'server.socket_port': PORT,
-            'server.socket_host': '0.0.0.0',
-            'server.thread_pool': 10,
-            'server.max_request_body_size': 0
-        })
-
-        cherrypy.engine.start()
-        cherrypy.engine.block()
+        serve(APP)
```

Comment:
```suggestion
        serve(APP, config={'server.max_request_body_size': 0, 'server.socket_timeout': 60})
```
to be able to transfer files >100Mb size